Demo/UITests/BraintreeDropIn_UITests.swift

```diff
@@ -993,8 +993,8 @@ class BraintreeDropIn_ThreeDSecure_VaultedPaymentMethod_UITests: XCTestCase {
     }
 
     func testDropIn_threeDSecure_2_challengeFlow_withVaultedPaymentMethod() {
-        waitForElementToBeHittable(app.collectionViews.cells.firstMatch)
-        app.collectionViews.cells.firstMatch.tap()
+        waitForElementToAppear(app.tables.firstMatch.staticTexts["Visa"])
+        app.cells.firstMatch.staticTexts["Visa"].tap()
```

Comment:
Our UI tests can't find the collection view now that it's inside a table view. But this seems to work.